api/src/test/java/org/cardanofoundation/rosetta/common/util/ValidateParseUtilTest.java

```diff
@@ -0,0 +1,92 @@
+package org.cardanofoundation.rosetta.common.util;
+
+import org.openapitools.client.model.Operation;
+import org.openapitools.client.model.PublicKey;
+import org.openapitools.client.model.VoteRegistrationMetadata;
+
+import org.junit.jupiter.api.Test;
+
+import org.cardanofoundation.rosetta.common.exception.ApiException;
+
+import static org.cardanofoundation.rosetta.EntityGenerator.givenPublicKey;
+import static org.cardanofoundation.rosetta.common.util.ValidateParseUtil.validateAndParsePoolKeyHash;
+import static org.cardanofoundation.rosetta.common.util.ValidateParseUtil.validateAndParseTransactionInput;
+import static org.cardanofoundation.rosetta.common.util.ValidateParseUtil.validateAndParseVoteRegistrationMetadata;
+import static org.cardanofoundation.rosetta.common.util.ValidateParseUtil.validateAndParseVotingKey;
+import static org.cardanofoundation.rosetta.common.util.ValidateParseUtil.validateChainCode;
+import static org.cardanofoundation.rosetta.common.util.ValidateParseUtil.validateDnsName;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.openapitools.client.model.CurveType.EDWARDS25519;
+
+@SuppressWarnings("java:S5778")
+class ValidateParseUtilTest {
```

Comment:
If no positive scenarios will be here (are they covered by other classes?) then let's rename it to make an intention clear.